monitoring-bot/internal/bridge.go

```
@@ -0,0 +1,152 @@
+package internal
+
+import (
+	"fmt"
+	"math/big"
+
+	"github.com/stellar/go/clients/horizonclient"
+	"github.com/stellar/go/keypair"
+	strNet "github.com/stellar/go/network"
+	"github.com/stellar/go/txnbuild"
+	client "github.com/threefoldtech/tfchain/clients/tfchain-client-go"
+)
+
+func (m *Monitor) monitorBridges() error {
+	message := "TfChain Bridges to Stellar Monitor Results:\n"
+	for _, net := range networks {
+
+		res, err := m.monitorBridge(net)
+		if err != nil {
+			message += fmt.Sprintf("\n - %s\n%s\n\n", res, err.Error())
+			fmt.Println(err.Error())
+			continue
+		}
+		message += fmt.Sprintf("\n - %s\n\n", res)
+	}
+
+	if err := m.sendBotMessage(message); err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func (m *Monitor) monitorBridge(net network) (string, error) {
```

I think we should return the error and check which msg to report in `monitorBridges`